django/db/models/fields/__init__.py

```diff
@@ -113,6 +113,8 @@ class Field(RegisterLookupMixin):
                              "%(date_field_label)s %(lookup_type)s."),
     }
     class_lookups = default_lookups.copy()
+    deprecated_for_migrations = False
```

Comment:
So, values of deprecated_for_migrations can be True, False and dict, right? I think this is a bit complicated. I'd set the default value to None and only accept a dict.